### PR TITLE
Feat: Added cards

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Added Cards: Vertical, Horizontal and text-only card

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

File: utils/card.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+.card {
+    display: flex;
+    flex-direction: column;
+    max-width: 300px;
```
Comment (Raghav888) [inline utils/card.css]: Use rem

File: utils/card.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    background: #ffffff;
+}
+
+.card_image {
+    height: 200px;
```
Comment (Raghav888) [inline utils/card.css]: Change all values to rem

**Review by Raghav888 — APPROVED** [review]
If the value is less than 10px, then only use PX.

File: utils/card.css (hunk 1 above)
Comment (munjalSaksham71) [reply utils/card.css]: This is the exact size of the card, thats why i gave it in px. i dont want it to be responsove or so.

File: utils/card.css (hunk 2 above)
Comment (munjalSaksham71) [reply utils/card.css]: Same here, The Exact size of card

**Pull request merged**